### ISSUE TITLE
Pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified)

### ISSUE THREAD

**Issue by CodeEagle** [RNCryptor/RNCryptor#166]
Pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified)

**Comment by rnapier** [comment]
This appears to be a bug in Xcode. http://www.openradar.me/22917151

I don't know of any work around until Xcode is fixed.

**Comment by karantza** [comment]
Just to add to this, I've noticed it in my project too. I've got warnings-as-errors turned on, so it was actually causing the build to fail. I tried a clang diangnostic pragma around my @import statement, but that doesn't seem to work like it does on #import or #include. 
The only workaround I found was to add -Wno-nullability-completeness to my host project's Other Warning Flags section. Of course, this turns off that warning for the whole app, which is unfortunate, but it's better than nothing.

**Comment by rnapier** [comment]
Update: This issue continues in the the final 7.2. Haven't discovered any way to address it yet.

**Comment by codywinton** [comment]
Showing up for me as well:

<img width="293" alt="screen shot 2016-01-19 at 3 37 32 pm" src="https://cloud.githubusercontent.com/assets/4325900/12432791/f762c27a-bec2-11e5-9504-0456aaea1e9b.png">

**Comment by rnapier** [comment]
To reiterate for future commenters: is a known bug in Swift (radar://22805286). Apple has not responded with a fix or workaround. If you discover a workaround or fix, please post. If you just want to keep track of it, you can click "Subscribe" in the right-hand pane.

**Comment by chadmoone** [comment]
FWIW, marking the file (or even just that line) as an audited region will prevent the warning:
```objc
NS_ASSUME_NONNULL_BEGIN
static NSString * const RNCryptorErrorDomain = @"RNCryptor.RNCryptorError";
NS_ASSUME_NONNULL_END
```

**Comment by rnapier** [comment]
This should now be resolved in 5.0. I've moved ObjC bridging to RNCryptor/RNCryptor-objc.

**Comment by hrithikpurwar** [comment]
Remove all headerfiles from /usr/local/include that have shortcut icons on them.
Worked as permanent fix for me.
